importer/handlers/utils.py

```diff
@@ -24,3 +26,18 @@ def should_be_imported(layer: str, user: get_user_model(), **kwargs) -> bool:
         return False
     
     return True
+
+
+def create_alternate(layer_name, execution_id):
+    '''
+    Utility to generate the expected alternate for the resource
+    is alternate = layer_name_ + md5(layer_name + uuid)
+    '''
+    _hash = hashlib.md5(
+        f"{layer_name}_{execution_id}".encode('utf-8')
+    ).hexdigest()
+    alternate = f"{layer_name}_{_hash}"
+    if len(alternate) >= 64: # 64 is the max table lengh in postgres
```

Comment:
if 64 is ok, let's check only for ">", not ">="

Reply:
the max lengh is 63, so or we keep this or it should be changed into: >63